server/cp/ingest/parser/cp_onclusive.py

```diff
@@ -21,7 +21,7 @@ class CPOnclusiveFeedParser(OnclusiveFeedParser):
     def parse(self, content, provider=None):
         onclusive_cv_items = _get_cv_items("onclusive_ingest_categories")
         anpa_categories = _get_cv_items("categories")
-        event_types_cvs = _get_cv_items("event_types")
+        event_types_cvs = _get_cv_items("CP_event_types")
```

Comment:
we should only use active items, so would be good to check `is_active` flag, and add it to vocabularies.json